pkg/registry/search/storage/search.go

```diff
@@ -50,7 +54,7 @@ func (r *SearchREST) NamespaceScoped() bool {
 
 // ConnectMethods returns the list of HTTP methods handled by Connect.
 func (r *SearchREST) ConnectMethods() []string {
-	return []string{"GET"}
+	return proxyMethods
```

Comment:
Does this mean `search` api starts to support `"POST", "PUT", "PATCH", "DELETE", "HEAD", "OPTIONS"`?

Reply:
Proxy supports all the methods, so I add them here.
If the original search not support these methods,  shall add logic
```go
if sets.NewString("POST", "PUT", "PATCH", "DELETE", "HEAD", "OPTIONS").Has(method) {
    return apierrors.NewMethodNotSupported(gvr, method)
}
```